test/support/example.bin

```diff
@@ -0,0 +1 @@
+	
```

Comment:
i think it would be better to use the  "test\expectdata" folder for that - so why adding a new folder?